README.md

```diff
@@ -1,64 +1,51 @@
-# Tower CLI app
+## Tower CLI
 
-Proof of concept of Tower command line client app based on Java native compilation, Micronaut HTTP declarative client and Tower domain classes
+Tower command line client allows to manage all the resources available at Tower. 
 
-## Installation
+### Installation
 
-From the last Github action at artifacts section, download the binary for your OS. Unzip it, give it execution permissions with `chmod +x ./tw` and move it into a folder that it's in your path (ex: `sudo mv ./tw /usr/local/bin/tw`)
+Tower CLI it is distributed as a single binary exacutable file. You only need to download your platform binary and execute it.
```

Comment:
- Tower CLI ~it~ is distributed
- ~You only need to download your platform binary and execute it.~ Just execute it after downloading your platfom binary